Updating using two editors docs (#15925)
ckeditor/ckeditor5: docs/installation/advanced/using-two-editors.md

```bash
git clone -b stable git@github.com:<your-username>/ckeditor5.git
cd ckeditor5/packages/ckeditor5-build-classic
yarn install
```

Now it is time to add the missing editor package and install it:

```bash
yarn add -D @ckeditor/ckeditor5-editor-inline
```

> Once all the dependencies are installed, you will need to modify the `src/ckeditor.ts` file, which currently only exports a single class. The first step is to move all plugins and configuration to variables so they can be reused by both editors:

Reviewer: ```suggestion
After installing all the dependencies, modify the `src/ckeditor.ts` file, which currently only exports a single class. The first step is to move all plugins and configuration to variables so they can be reused by both editors:
```

Maybe we could avoid passive voice?

```js
import { ClassicEditor as ClassicEditorBase } from '@ckeditor/ckeditor5-editor-classic';

// Other imports

class ClassicEditor extends ClassicEditorBase {}

const plugins = [
  // ...
]

const config = {
  // ...
}

ClassicEditor.builtinPlugins = plugins;
ClassicEditor.defaultConfig = config;

export {
  ClassicEditor
};
```

> Now you can add the `InlineEditor` class to the file, add the same plugins and configuration to it and export it:

Reviewer: ```suggestion
Now, you can add the `InlineEditor` class to the file. Add the same plugins and configuration to the class and then export it:
```

I would change the punctuation a little.
